pkg/kubectl/cmd/taint.go

```diff
@@ -76,16 +75,6 @@ var (
 func NewCmdTaint(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	options := &TaintOptions{}
 
-	// retrieve a list of handled resources from printer as valid args
```

Comment:
> @deads2k is this done for a reason here?

I hadn't noticed this code before.  It's definitely incorrect for this command and its not a very clean way of getting the data for the completions overall.  Removing it here is good.